packages/affiliates/apps/GenerateDevMiningPr.js

```diff
@@ -0,0 +1,44 @@
+require("dotenv").config();
+const { makeUnixPipe, devMiningPrTemplate } = require("../libs/affiliates/utils");
+
+// This expects to be piped json input which contains this structure:
+// {
+//   "issueNumber": 2620,
```

Comment:
where does the issue number come from?

Reply:
from [apps/OpenDevMiningIssue](https://github.com/UMAprotocol/protocol/blob/master/packages/affiliates/apps/OpenDevMiningIssue.js#L23)